mapping/sustainability.yaml

```
@@ -323,7 +323,7 @@
     mapping: |-
         Project level:
 
-        For each measure add a `ClimateMeasure` object to the `environment.climateMeasures` array, setting it's `.type` according to the matching code in the open climateMeasuresType codelist and mapping the further explanation to its `.description`.
+        For each measure add a `ClimateMeasure` object to the `environment.climateMeasures` array, setting it's `.type` according to the matching code in the open climateMeasures codelist and mapping the further explanation to its `.description`.
```

Use simple rather than continuous tense. Add a link to the climate measures codelist.

```suggestion
        For each measure, add a `ClimateMeasure` object to the `environment.climateMeasures` array, set its `.type` using the open [climateMeasures](../../reference/codelists.md#climateMeasures) codelist and map any further explanation to its `.description`.
```